zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -262,21 +262,21 @@ pub fn init_wallet_db<P: consensus::Parameters + 'static>(
     init_wallet_db_internal(wdb, seed, &[], true)
 }
 
-fn init_wallet_db_internal<P: consensus::Parameters + 'static>(
+pub(crate) fn init_wallet_db_internal<P: consensus::Parameters + 'static>(
```

Comment:
AFAICT this is unnecessary; the new places we use this are all in sub-modules. So to ensure we don't use this elsewhere in the crate, undo this:
```suggestion
fn init_wallet_db_internal<P: consensus::Parameters + 'static>(
```